govcd/nsxt_alb_importable_clouds.go

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2021 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+// NsxtAlbImportableCloud allows user to list importable NSX-T ALB Clouds. Each importable cloud can only be imported
+// once. It has a flag AlreadyImported which hints if it is already consumed or not.
+type NsxtAlbImportableCloud struct {
+	NsxtAlbImportableCloud *types.NsxtAlbImportableCloud
+	vcdClient              *VCDClient
+}
+
+// GetAllAlbImportableClouds returns importable NSX-T ALB Clouds.
+// ID (URN format) of parent ALB Controller is mandatory
```

Comment:
Maybe this way it's clearer?
```suggestion
// parentAlbControllerUrn (ID in URN format of a parent ALB Controller) is mandatory
```

Reply:
Changed